core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -1858,6 +1859,14 @@ private long loadFileMetadataAndJournal(LockedInodePath inodePath,
     CreateFileOptions createFileOptions =
         CreateFileOptions.defaults().setBlockSizeBytes(ufsBlockSizeByte)
             .setRecursive(options.isCreateAncestors()).setMetadataLoad(true).setPersisted(true);
+    if (SecurityUtils.isSecurityEnabled(MasterContext.getConf())) {
+      String ufsOwner = ufs.getOwner(ufsUri.toString());
+      String ufsGroup = ufs.getGroup(ufsUri.toString());
+      short ufsPermission = ufs.getMode(ufsUri.toString());
+      createFileOptions = createFileOptions.setPermission(
+          new Permission(ufsOwner, ufsGroup, ufsPermission));
+    }
```

Comment:
Good point. `isSecurityEnabled` is not necessary because we can still set permission on `inode` when security is disabled.
updated, thanks!